Statistics/Statistics.cs

```diff
@@ -1,12 +1,41 @@
-﻿using System;
+using System;
+using Statistics;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace Statistics
 {
-    public class StatsComputer
+    public class Stats
     {
-        public Stats CalculateStatistics(List<float> numbers) {
-            //Implement statistics here
-        }
+        public double average;
+        public double max;
+        public double min;
     }
+    
+    public class StatsComputer{ 
+        public Stats CalculateStatistics(List<double> numbers) {
+            
+         
+            if(numbers.Count == 0){
+            Stats myStats = new Stats();
+                
```

Comment:
What about including the NaN initialization in the Stats constructor itself? Then you don't have to worry about initializing in every place where you need in future...